src/index.html

```diff
@@ -5,8 +5,572 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+      crossorigin = "crossorigin">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <div class="catalog">
+      <header
+        class="catalog__header header">
+        <a
+          href="#home"
+          class="header__logo">
+          <img
+            src="images/logo.png"
+            alt="Moyo"
+            class="header__logo-img"
+          >
+        </a>
+        <nav
```

Comment:
```suggestion
        </a>
        
        <nav
```

Reply:
Please, fix similar mistake in all other places
<img width="1092" alt="image" src="https://github.com/mate-academy/layout_catalog/assets/86789161/fa30ade4-3af5-4646-9a90-a6920139319b">